anchore_engine/db/entities/policy_engine.py

```diff
@@ -365,7 +364,7 @@ def get_nvd_identifiers(self, _nvd_cls, _cpe_cls):
                     if cve_id and cve_id not in cves:
                         cves.append(cve_id)
         except Exception as err:
-            log.warn(
+            log.warning(
```

Comment:
log.warn() is used all over the place and needs to be updated. But it should be handled in a different commit as it's unrelated to this change